src/zoid/card-fields/component.jsx

```diff
@@ -51,6 +51,9 @@ type CardFieldsProps = {|
   style?: {|
     height: number,
   |},
+  styleOptions?: {|
+    disablePrerender?: boolean,
```

Comment:
Should this be something like `disableMaxHeight` so that it is consistent with the buttons style options? I'm not sure though, cc @wsbrunson

Reply:
are you asking if it should be in the `style` object?

Reply:
Oh actually I got flags confused, disregard